txpipe/twopoint_fourier.py

```diff
@@ -484,19 +606,19 @@ def select_calculations(self, nbins_source, nbins_lens):
             for j in range(i + 1):
                 calcs.append((i, j, k))
 
-        # For shear-position we use all pairs
-        k = SHEAR_POS
-        for i in range(nbins_source):
-            for j in range(nbins_lens):
-                calcs.append((i, j, k))
+#         # For shear-position we use all pairs
```

Comment:
We need to keep these calculations

Reply:
oops, yes I'll uncomment this and the others below. I think we should add config-based selection of the calculations later as the deprojection can be slow.